src/core/issues-notes.js

```diff
@@ -146,6 +159,33 @@ function handleIssues($ins, ghIssues, conf) {
   }
 }
 
+//derives the text-color-class based on the illumination score of the labelColor
+function deriveTextColorClass(hexColor) {
+  const rgb = parseInt(hexColor, 16);
+  const red = (rgb >> 16) & 0xff;
+  const green = (rgb >> 8) & 0xff;
+  const blue = (rgb >> 0) & 0xff;
+  const illumination = 0.2126 * red + 0.7152 * green + 0.0722 * blue;
+  return illumination > 140 ? "light" : "dark";
+}
+
+function createLabels(label) {
+  const hexColor = label.color.substring(1);
+  var textColorClass = "dark";
+  var labelElement = document.createElement("a");
+  if (!isNaN(parseInt(hexColor, 16))) {
```

Comment:
parseInt is being done twice: here and in `deriveTextColorClass`. You should only do it in  `deriveTextColorClass()` and throw if NaN. Then here, you should deal with that exception.

Reply:
Yep got it